app.js

```diff
@@ -32,6 +32,8 @@ process.on("exit", () => {
     lib.destroy();
 });
 
+app.use(express.json());

```

Comment:
I think the raw parse can work even better (no need to stringify and handles different data types) https://expressjs.com/en/api.html#express.raw



https://thewebdev.info/2021/07/12/how-to-get-raw-request-body-with-express/

Reply:
Using [express.raw](https://expressjs.com/en/api.html#express.raw) which parses request's payloads into a buffer.